lib/nerves_wpa_supplicant.ex

```diff
@@ -82,7 +82,7 @@ defmodule Nerves.WpaSupplicant do
       Option                | Description
       ----------------------|------------
       :ssid                 | Network name. This is mandatory.
-      :key_mgmt             | The security in use. This is mandatory. Set to :NONE, :WPA_PSK
+      :key_mgmt             | The security in use. This is mandatory. Set to NONE or WPA-PSK
```

Comment:
I think we should be explicit that the "right" thing to do is either `:NONE` or `"WPA-PSK"`, even though `"NONE"` and `:"WPA-PSK"` would also work.